Updated Jenkins docs (#13506)
newrelic/docs-website: src/content/docs/more-integrations/open-source-telemetry-integrations/jenkins/monitoring-jenkins-ot.mdx

1. Log into Jenkins.
2. From the Dashboard, click on **Manage Jenkins**.
3. Under System Configuration, click **Plugins**.

> 4. Click the **Available plugins** tab, and in the search bar, type **OpenTelemetry**.

Reviewer: ```suggestion
4. Click the **Available plugins** tab, then search for **OpenTelemetry**.
```
